README.md

```diff
@@ -4,26 +4,23 @@
 [![Docs](https://img.shields.io/badge/api-docs-green.svg?style=flat)](https://hexdocs.pm/websock)
 [![Hex.pm](https://img.shields.io/hexpm/v/websock.svg?style=flat&color=blue)](https://hex.pm/packages/websock)
 
-WebSock is a specification for apps to service WebSocket connections; you can think
-of it as 'Plug for WebSockets'. WebSock abstracts WebSocket support from servers such as
-[Bandit](https://github.com/mtrudel/bandit/) or [Cowboy](https://github.com/ninenines/cowboy)
-and exposes a generic WebSocket API to applications. WebSocket-aware
-applications such as Phoenix can then be hosted within a supported web server
-simply by defining conformance to the `WebSock` behaviour, in the same manner as
-how Plug conformance allows their HTTP aspects to be hosted within an arbitrary
-web server.
+WebSock is a specification for apps to service WebSocket connections; you can
+think of it as 'Plug for WebSockets'. WebSock abstracts WebSocket support from
+servers such as [Bandit][] or [Cowboy][] and exposes a generic WebSocket API to
+applications. WebSocket-aware applications such as Phoenix can then be hosted
+within a supported web server simply by defining conformance to the `WebSock`
+behaviour, in the same manner as how Plug conformance allows their HTTP aspects
+to be hosted within an arbitrary web server.
 
-This package defines the `WebSock` behaviour which describes the functions that
-an application such as Phoenix must implement in order to be WebSock compliant; it
-is roughly the equivalent of the `Plug` interface, but for WebSocket
-connections. It is commonly used in conjunction with the
-[websock_adapter](https://hex.pm/packages/websock_adapter) package which
-defines concrete adapters on top of [Bandit](https://github.com/mtrudel/bandit/)
-and [Cowboy](https://github.com/ninenines/cowboy); the two packages are separate
-to allow for servers which directly expose `WebSock` support to depend on just
-the behaviour. Users will almost always want to depend on
-[websock_adapter](https://hex.pm/packages/websock_adapter) instead of this
-package.
+<!-- MDOC -->
```

Comment:
Out of curiosity what is this comment about?

Reply:
It's a marker to know what things we want to extract for example, MDOC is sort of an abbreviation for Markdown and the marker is "common" in the community, AFAIK is in a couple of projects like `oban`, `elixir-nx/explorer`, etc.

Reply:
Interesting! Is it purely for human consumption or is it machine readable?